### PR TITLE
Finish open sourcing

### PR DESCRIPTION
**Pull request by tmathern**
- Removed `.circleci` directory
- Update package name and version
-  Update dependencies as needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,25 +1,22 @@
 {
-  "name": "@nui/node-cloudstorage",
-  "version": "5.0.0",
+  "name": "@adobe/cloudstorage",
```
Comment (alexkli) [inline package.json:2]: I think we should use a more specific name here. This does sound pretty broad, or one might assume it's some form of Adobe platform storage.

It is focused on
- _blob_ storage
- thin wrapper for presigned urls, upload, put & listing

Maybe `@adobe/cloud-blob-storage`?

cc @DrGirlfriend @jdelbick @bobvanmanen

Comment (tmathern) [reply package.json]: I changed the name to `@adobe/cloud-blob-storage`.

@alexkli Can you rename the repo so the names fit?

Comment (tmathern) [reply package.json]: @alexkli How do I use credentials in the tests? Are they in a file? Do I access them through an env variable?

Current error: `INFO: AZURE_STORAGE_KEY and/or AZURE_STORAGE_ACCOUNT are not set, trying file: /home/travis/.adobe-asset-compute/credentials.yaml`

Comment (alexkli) [reply package.json]: Let's get feedback from the group. I am not sure `@adobe/cloud-blob-storage` is good either...

**Comment by alexkli** [comment]
@tmathern (separate thread) they need to be env vars configured in travis. can take of that later today.

File: .eslintrc.js [hunk 2]
```diff
@@ -11,5 +11,5 @@ governing permissions and limitations under the License.
 */
 
 module.exports = {
-    "extends": "@adobe/eslint-config-asset-compute"
-};
\ No newline at end of file
+    "extends": "@adobe/eslint-config-asset-compute",
```
Comment (alexkli) [inline .eslintrc.js:14]: Trailing comma?

Comment (tmathern) [reply .eslintrc.js:14]: (Ignore that, putting it back and removing it was my change to retrigger tests).

**Comment by tmathern** [comment]
Most .presign call are totally sync, as are the methods/functions they call. So I removed the unneeded `await`s in tests.

File: package.json (hunk 1 above)
Comment (tmathern) [reply package.json]: @pheenomenon @DrGirlfriend @jdelbick ^^^^^

Comment (DrGirlfriend) [reply package.json]: Plus Azure & Google call it blob and aws calls it object....
virtual-storage?
cloud-services?
cloud-service-storage?

This is going to be hard

Comment (tmathern) [reply package.json]: `cloud-binary-storage` ?

**Comment by tmathern** [comment]
(@jdelbick There already are keywords there. You don't see them in this PR, because keywords were added previously and they didn't change).

**Pull request merged**